src/index.jsx

```diff
@@ -82,97 +90,151 @@ class BerbixVerify extends React.Component {
+    const {
+      overrideUrl,
+      version,
+      clientId,
+      role,
+      templateKey,
+      email,
+      phone,
+      continuation,
+      clientToken,
+      showInModal
+    } = this.props;
     if (overrideUrl != null) {
       return overrideUrl;
     }
     const token = clientToken || continuation;
     const template = templateKey || role;
-    var options = ['sdk=BerbixReact-' + SDK_VERSION];
+    var options = ["sdk=BerbixReact-" + SDK_VERSION];
     if (clientId) {
-      options.push('client_id=' + clientId);
+      options.push("client_id=" + clientId);
     }
     if (template) {
-      options.push('template=' + template);
+      options.push("template=" + template);
     }
     if (email) {
-      options.push('email=' + encodeURIComponent(email));
+      options.push("email=" + encodeURIComponent(email));
     }
     if (phone) {
-      options.push('phone=' + encodeURIComponent(phone));
+      options.push("phone=" + encodeURIComponent(phone));
     }
     if (token) {
-      options.push('client_token=' + token);
+      options.push("client_token=" + token);
     }
-    return (this.baseUrl() + '/' + version + '/verify?') + options.join('&');
+    if (showInModal) {
+      options.push("modal=true");
+    }
+    return this.baseUrl() + "/" + version + "/verify?" + options.join("&");
   }
 
   render() {
     if (!this.state.show) {
       return null;
     }
-    return (
+    const iframe = (
       <iframe
         key={this.state.idx}
         src={this.frameUrl()}
         style={{
-          backgroundColor: 'transparent',
-          border: '0 none transparent',
+          backgroundColor: "transparent",
+          border: "0 none transparent",
           padding: 0,
           margin: 0,
-          display: 'block',
-          width: '100%',
-          height: this.state.height + 'px',
-          overflow: 'hidden',
+          display: "block",
+          width: "100%",
+          height: this.state.height + "px",
+          overflow: "hidden"
         }}
         allow="camera"
         scrolling="no"
         referrerPolicy="no-referrer-when-downgrade"
       />
     );
+    if (this.props.showInModal) {
+      const disp = Math.max(this.state.height, window.innerHeight);
+      const marginTop = disp > 500 ? (disp - 500) * 0.25 : 10;
+      return (
+        <div
+          className="--berbix-overlay"
+          style={{
+            width: "100%",
+            height: "100%",
+            position: "fixed",
+            top: 0,
+            left: 0,
+            right: 0,
+            bottom: 0,
+            backgroundColor: "rgba(0, 0, 0, 0.6)",
+            zIndex: 1000
+          }}
+        >
+          <div
+            style={{
+              margin: "0 auto",
+              top: "100px",
+              width: "100%",
+              maxWidth: "500px",
+              maxHeight: "100%",
+              overflow: "auto",
+              borderRadius: "5px",
```

Comment:
Is this intentional?

Reply:
Hm, nice catch. Leftover from a previous attempt.